containers/enterprise.go

```diff
@@ -1,15 +1,29 @@
 package containers
 
-import "dagger.io/dagger"
+import (
+	"dagger.io/dagger"
+)
 
 const BusyboxImage = "busybox:1.36"
 
 func InitializeEnterprise(d *dagger.Client, grafana *dagger.Directory, enterprise *dagger.Directory) *dagger.Directory {
-	return d.Container().From(BusyboxImage).
+	hash := d.Container().From(GitImage).
+		WithDirectory("/src/grafana-enterprise", enterprise).
+		WithWorkdir("/src/grafana-enterprise").
+		WithEntrypoint([]string{}).
+		WithExec([]string{"/bin/sh", "-c", "git rev-parse --short HEAD > .enterprise-commit"}).
+		File("/src/grafana-enterprise/.enterprise-commit")
+
+	// Initializes Grafana Enterprise in the Grafana directory
+	g := d.Container().From(BusyboxImage).
 		WithDirectory("/src/grafana", grafana).
 		WithDirectory("/src/grafana-enterprise", enterprise).
 		WithWorkdir("/src/grafana-enterprise").
+		WithFile("/src/grafana/.enterprise-commit", hash).
```

Comment:
That file should probably be deleted before packaging, right?

Reply:
packaging uses an explicit list of paths to package so it won't be included.